### ISSUE TITLE
Font-Face Load Detection not Working in Firefox

### ISSUE THREAD

**Issue by jschlieber** [zachleat/fontfaceonload#25]
Font-Face Load Detection is not working in Firefox (tested in 41.0.1 and 41.0.2), i.e. always ending up in the error function (_fontfaceonload version 0.1.6_). The same test works on Chrome and IE. I used the following to test:

**HTML**

``` html
<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta http-eqiv="X-UA-Compatible" content="IE=edge,chrome=1">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="font-face.css">
  <title>Font Face Test</title>
</head>
<body>
  <p>Normal Text</p>
  <div class="load-font">
    <p>Impact Text</p>
  </div>
<script src="fontfaceonload.js"></script>
<script src="font-face-test.js"></script>
</body>
```

**font-face.css**

``` css
@font-face {
  font-family: 'impactreg';
  font-style: normal;
  font-weight: 400;
  src:  url('/font/impactreg.eot'); /* IE9 Compat Modes */
  src:  local('impactreg'),
        url('/font/impactreg.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
        url('/font/impactreg.woff') format('woff'),
        url('/font/impactreg.ttf') format('truetype');
}

.load-font {
  font-family: 'impactreg', sans-serif;
}
```

**font-face-test.js**

``` javascript
FontFaceOnload('impactreg', {
  success: function() {
    alert('Success!');
  },
  error: function() {
    alert('Error!');
  }
});
```

The font used in the example can be found here: http://fontsforweb.com/font/generatezip/?id=7349

**Comment by jschlieber** [comment]
Ok, just noticed it should be fixed in 0.1.7, but i can't install it using bower (*No tag found that was able to satisfy ~0.1.7*)

**Comment by zachleat** [comment]
New versions have been published since 2015 😭😱
